var/spack/repos/builtin/packages/libcatalyst/package.py

```diff
@@ -12,9 +12,12 @@ class Libcatalyst(CMakePackage):
 
     homepage = "https://gitlab.kitware.com/paraview/catalyst"
     git = "https://gitlab.kitware.com/paraview/catalyst.git"
+    url = "https://gitlab.kitware.com/api/v4/projects/paraview%2Fcatalyst/packages/generic/catalyst/v2.0.0/catalyst-v2.0.0.tar.gz"
 
     maintainers = ["mathstuf"]
 
+    version("2.0.0-rc3", sha256="8862bd0a4d0be2176b4272f9affda1ea4e5092087acbb99a2fe2621c33834e05")
```

Comment:
how about we specify this as a git revision rather than a url?

Reply:
For what purpose? Why is cloning better than a tarball?

Reply:
We can skip having both url|git property here

Reply:
Before the auto-merge we have this comment opened too. I am no expert on this but I wonder what would be the gain of adding two urls (git|http URL) for a package? with a git url we can also specify git tags.

Reply:
I don't think asking people to contact `gitlab.kitware.com` for this package rather than being able to cache a tarball would work long-term for some HPC deployments…

Reply:
Fair enough